backend/uclapi/common/pagination.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import pagination
+from rest_framework.response import Response
+
+
+class CustomPagination(pagination.PageNumberPagination):
```

Comment:
`CursorPagination` might be possible however we would need an additional created column in the table and given how rarely the data updates I am not sure if it is entirely necessary. It is also harder to switch as the `CursorPagination` class does not have the same request path which we rely on to find the module name.

Reply:
I don't think we would, as I've mentioned in a different comment, if we make `sensor_id` and `survey_id` compulsory arguments to the endpoint, then at the database level we'd have a unique ordering via the `datetime` field, which makes us eligible to use it and benefit from it's asymptotic properties. Note that the table doesn't need to have a `created` column, that's just the default.

The only thing I have doubts about is that `datetime` isn't globally unique, it's only unique together with `survey_id` and `sensor_id`. I believe it's still possible to do cursor pagination if you have composite keys, but it may be hard to get DRF's `CursorPagination` to accept that...

Indeed, the data updates rarely now, all the libraries are closed...

The pagination speed may be fast now, but the dataset is only going to grow larger and it's going to be hard to change the API later if performance does degrade.

1) First see how easy it is to get `CursorPagination` working (if at all).
2) Otherwise, benchmark the performance now, and if it's so good that even future data is not a problem, we'll keep the code as is.